frontend/src/sass/pages/main-landing.scss

```diff
@@ -116,7 +116,7 @@
   }
 }
 
-#help-pick-background{
+#help-me-pick-background{
```

Comment:
The ID in the HTML wasn't changed. Does this change cause the styling to go sideways?

Reply:
@mgwalker - From everything I can see, there is no corresponding styling. So no, I haven't seen any wonkiness as result

Reply:
`$("#help-pick-background")` from the landing page does return an element:

![image](https://user-images.githubusercontent.com/1775733/96278298-f386a580-0f9a-11eb-9ecf-ed7e862b5cee.png)

If I then change its ID to `help-me-pick-background`, it changes to this:

![image](https://user-images.githubusercontent.com/1775733/96278362-0305ee80-0f9b-11eb-93be-2beb633e874d.png)

Reply:
That is totally my bad. I did my search incorrectly. I updated the html accordingly

Reply:
For further clarification, that ID #help-pick-background was causing the AMP checks to fail. That's why I needed to change it. Somewhere along the way I changed the html ID back again.

![image](https://user-images.githubusercontent.com/23467847/96285878-259d0500-0fa5-11eb-8ebe-573d58287acb.png)